utils/create_dummy_models.py

```diff
@@ -974,6 +974,10 @@ def get_token_id_from_tokenizer(token_id_name, tokenizer, original_token_id):
 
 
 def get_config_overrides(config_class, processors):
+    # `Bark` configuration is too special. Let's just not handle this for now.
+    if config_class.__name__ == "BarkConfig":
+        return {}
+
```

Comment:
Out of curiosity, what is the purpose of the function `get_config_overrides` ?

Reply:
This is tiny model creation - which somehow messy. But `get_config_overrides` here is to use the values from the tokenizer in the config, like `vocab_size`. (note, this is not real model, and during creation, we use dummy values, but still need some consistency between tokenizer (or processor) and config)